src/server/channel_store.cc

```diff
@@ -124,8 +124,9 @@ unsigned ChannelStore::SendMessages(std::string_view channel, facade::ArgRange m
 
   // Make sure none of the threads publish buffer limits is reached. We don't reserve memory ahead
   // and don't prevent the buffer from possibly filling, but the approach is good enough for
-  // limiting fast producers. Most importantly, we can use DispatchBrief below as we block here
```

Comment:
I actually dont understand the comment that was removed.. what is "we block here"

Reply:
I removed it because I replaced DispatchBrief with AwaitBrief

Reply:
but ignore, what is important is to the semantic right. I will adjust the comment